### ISSUE TITLE
Lake Elrendar mobs

### ISSUE THREAD

**Issue by Kezeal** [Valhalla-Gaming/Tracker#5261]
In Ghostlands, the Lake Elrendar mobs seem extremely squishy.

**Comment by Tregeta** [comment]
Can you provide links so we can check these in the database?

**Comment by Kezeal** [comment]
http://www.wowhead.com/npc=16327/ravening-apparition
http://www.wowhead.com/npc=16328/vengeful-apparition
When I played as a mage in retail MoP, I remember these guys taking multiple hits, but here I overkilled them hard with a single frostbolt.

**Comment by Tregeta** [comment]
Were you higher level than these creatures by a good margin? 5-6 levels perhaps?

**Comment by Kezeal** [comment]
I was level 14 at the time, so only 2-3 levels. If my memory serves, I did the quests in the same order as in retail(the town ones until they got too difficult, then headed to the lake).

**Comment by Tregeta** [comment]
They appear to have the correct HP values @ 261 for level 11 and 312 for level 12. I am quite surprised that your frostbolt can one shot these creatures at this level

**Comment by Kezeal** [comment]
I was doing too much damage compared to the mobs around the lake(even the trolls that were level 10 or 11 sometimes needed more than 1 hit at that level). Perhaps they have an issue with defenses/resistances?

**Comment by Tregeta** [comment]
Seems normal damage wise if you have heirlooms. Do you have any or a full set?

**Comment by Kezeal** [comment]
No, it was my first character. Just quest-given and mob-looted items.

**Comment by Tregeta** [comment]
It's unlikely you have the legacy buff (when creatures are level 11-12) if you were only level 14 unless you are mistaken about the level.

For reference the legacy buff scales your damage higher when facing mobs much lower than your current level.

Further research shows that damage values of these creatures and their health values match retail. It could be something with your class damage spells.
